keyboards/dmqdesign/spin/info.json

```
@@ -0,0 +1,27 @@
+{
+    "keyboard_name": "SPIN", 
+    "url": "https://www.DMQdesign.com", 
+    "maintainer": "Quarren42", 
+    "width": 4, 
+    "height": 4, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+            {"x":0, "y":0},
+            {"x":1, "y":0},
+            {"x":2, "y":0},
+            {"x":3, "y":0},
+            {"x":0, "y":1},
+            {"x":1, "y":1}, 
+            {"x":2, "y":1},
+            {"x":3, "y":1.5},
+            {"x":0, "y":2},
+            {"x":1, "y":2},
+            {"x":2, "y":2},
+            {"x":0, "y":3}, 
+            {"x":1, "y":3},
+            {"x":2, "y":3},
+            {"x":3, "y":3}]
```

Closer but not quite (though this is an issue with the page we provide for converting the KLE data).

```suggestion
            {"x":3.5, "y":3},
            {"x":0, "y":3}, 
            {"x":1, "y":3},
            {"x":2, "y":3}
            ]
```